argocd-monitoring/components/rbac/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
```

Comment:
It should not be a component, if this is the core reason for the application to be present. Each kustomize scheme should have a base which is overlayed with optional components pulled to the mix. Can you please move this to the base?

Reply:
moved it to base :+1: